### PR TITLE
Sponsors page updates

### PR DESCRIPTION
**Pull request by shonnadorsey**
- Added sponsor logos for Omaha Public Library, Flywheel and Interface with a short description of the company and sponsorship. Also added links to company websites for each button.
- Added a modal popup in the short description above the current sponsor logos with a sponsorship request form embedded from Wufoo. 
- Adjusted style for .sponsor in style.css to reflect fixed padding above page info text, button background and text colors to match site style and default logo image dimensions.
- Removed logo dimensions from individual images on the sponsors page.

### CODE REVIEW

File: sponsors.html [hunk 1]
```diff
@@ -62,7 +62,49 @@
...
 		  <!-- Three columns of text below the carousel -->
 		  <div class="row">
-				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code</p>
+		  		<br />
+		  		<br />
+				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code. 
+				<br />
+				Interested in donating resources to a club? Click <a href="#" data-toggle="modal" data-target="#sponsorModal">here</a>
+
+					<div class="modal fade sponsorModal" tabindex="-1" role="dialog" aria-labelledby="sponsorModal" aria-hidden="true" id="bannerformmodal">
```
Comment (sandikbarr) [inline sponsors.html]: The modal is not opening for me. Looks like your modal id should be sponsorModal.

File: sponsors.html [hunk 2]
```diff
@@ -62,7 +62,49 @@
...
+								</script>
+								</div>
+              						<div class="modal-footer">
+                						<button type="button" class="btn btn-blue">Submit</button>
+              						</div>          
+        						</div>
+        						</div>
+      						</div>
+    					</div>
+				to complete a short form and our fundraising coordinator will contact you for more details.</p>
```
Comment (sandikbarr) [inline sponsors.html]: Probably put the modal div element somewhere besides between the Click here and ...to complete a short form. It looks broken up in my browser.
screen shot 2016-07-10 at 7 41 04 pm

File: sponsors.html [hunk 3]
```diff
@@ -72,23 +114,23 @@
 				</div>
 
 				<div class="col-md-3">
-				  <img class="img-circle" src="data:image/gif;base64,R0lGODlhAQABAIAAAHd3dwAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw==" alt="Generic placeholder image" width="140" height="140">
+				  <img class="img-circle" src="img/sponsors/library.jpeg" alt="Omaha Public Library" width="140" height="140">
 				  <h2>Heading</h2>
```
Comment (sandikbarr) [inline sponsors.html:123]: Oops we need Omaha Public Library here. :)

Comment (shonnadorsey) [inline sponsors.html:123]: Updated via commit: 727c7c8

File: sponsors.html (hunk 2 above)
Comment (shonnadorsey) [inline sponsors.html]: Updated to button and adjusted text via commit: 727c7c8

File: sponsors.html (hunk 1 above)
Comment (shonnadorsey) [inline sponsors.html]: Updated modal id via commit: 727c7c8

**Pull request merged**